torch/autograd/forward_ad.py

```diff
@@ -0,0 +1,55 @@
+import torch
+from .grad_mode import _DecoratorContextManager
+
+from typing import Any
+
+# Global variable used to make the python API simpler to use
+_current_level = -1
```

Comment:
Note that the "global" level only exist in python (and we can add it to the c++ api later).